packages/connect-wallet-modal/README.md

```diff
@@ -31,12 +31,17 @@ const walletsModalDefaultConfig = getDefaultWalletsModalConfig();
 
 | Prop | Description |
 |------|-------------|
-| `buttonComponentsByConnectorId` | A map of `ConnectButton`s associated with a certain connector id or type. <br /> ***Default**: Get with `getDefaultWalletsModalConfig()`* |
+| `config` | *A config object, see below.*  |
 | `darkThemeEnabled?` | Set to true for the dark color theme. <br /> ***Default**: `false`* |
 | `buttonsFullWidth?` | Specify connector buttons to render one per row. It could be handy if you are planning to use only few wallets in your dapp. |
+
+#### Config configuration list
```

Comment:
It's divided because component props and the `config` are different structures. Config structure is a prop and it has it's own fields to configure. You cannot pass, for example, `walletsShown` just as a prop to the component, this value should be passed as part of config. Also you cannot pass `darkThemeEnabled` to the config structure, because this is a regular component prop.

But if you want to merge those tables into one, I could do it renaming config fields adding prefix `config.[fieldName]`. Check it out now.